src/error.rs

```diff
@@ -94,6 +98,7 @@ impl HandlerErrorKind {
         match self {
             HandlerErrorKind::Validation(_) => StatusCode::BAD_REQUEST,
             HandlerErrorKind::AdmServerError() => StatusCode::SERVICE_UNAVAILABLE,
+            HandlerErrorKind::AdmLoadError() => StatusCode::PARTIAL_CONTENT,
```

Comment:
The Contile integration code in Firefox currently only handles HTTP 200 and 204. Given a 206, it will still try parsing the JSON from the response, which basically would share the same code path as HTTP 500.

Does it make sense to return HTTP 204 here instead?

Reply:
Do you mind adding integration tests for this, @jrconlin? 🤖

Reply:
Returning a 204 is fine. I thought we had suggested 206 initially, but it doesn't matter from the server.

I'm not sure of the best way to trigger the test, mostly because this relies on an external failure state. I can see if I can construct a viable test around this, but it might be too artificial to be useful.

Reply:
The integration test suite supports adding a delay (like a `time sleep()`) to responses, that could trigger the `Populating` state and possibly a subsequent step in the same scenario could then trigger this case?

Reply:
My mistake, the second step is blocked on the first step finishing, so that wouldn't work